README.md

```diff
@@ -2,7 +2,7 @@
 
 This project provides Helm charts and basic examples for deploying Pega on Kubernetes. You will also need to download the required [installation kit](https://community.pega.com/knowledgebase/products/platform/deploy) from the Pega Community which includes rules and data to preload into your relational database. Deploying Pega on Kubernetes requires Pega Infinity 8.2 or newer.
```

Comment:
can you also fix this terminology so it's consistent with our version descriptions?
Pega Infinity 8.2 or newer
to Pega Infinity 8.2 and later.

Reply:
updated